ChangeLog

```diff
@@ -6,6 +6,58 @@ Changes
    * Add unit tests for AES-GCM when called through mbedtls_cipher_auth_xxx()
      from the cipher abstraction layer. Fixes #2198.
 
+= mbed TLS 2.16.0 branch released xxxx-xx-xx
```

Comment:
This looks like it lists the changes in Mbed TLS 2.16.0 without a release date, but actually it only takes some of the changes of 2.16.0. Please don't use a version number here.

We don't have a precedent for splitting unreleased changes into two sections based on their origin. I think it would be fine to have a single `x.xx.xx` section here with the changes from both branches. But if you really think there should be separate sections, please use a non-misleading presentation, e.g. `mbed TLS 2.14+01b34fb316a5d8c37244719be3f11ccf2ae85e41 unreleased`

Reply:
As explained in the commit message, we have two sections because Mbed Crypto has a ChangeLog entry already added to the x.xx.x version for a future release. 2.16.0 without a release date was used as the marker for a development version of 2.16.0 (pre-release). Seems like this explanation wasn't clear enough, was in a location not looked at, and the marker used was too confusing in isolation. Will use 2.14+01b34fb316a5 as the version marker.

Reply:
I know what you did, but I don't understand why you did it, and the `2.16.0` would have to stand on its own because it doesn't look like one would need to track down the commit that introduced it to realize that `2.16.0` does not in fact mean 2.16.0.

Reply:
I've added a commit to change the identifier to something hopefully more clear.